lib/active_shipping/carriers/usps.rb

```diff
@@ -238,9 +238,9 @@ def extract_event_details(node)
       
       timestamp = "#{node.at('EventDate').text}, #{node.at('EventTime').text}"
       event_code = node.at('EventCode').text
-      city = node.at('EventCity').text
-      state = node.at('EventState').text
-      zip_code = node.at('EventZIPCode').text
+      city = try_text(node.at('EventCity'))
```

Comment:
Can't you just do:
`node.at('EventCity').try(:text)`

I don't think you need to define the `try_text` method